docs/topics/db/transactions.txt

```diff
@@ -308,9 +308,13 @@ Pass any function (that takes no arguments) to :func:`on_commit`::
 
     transaction.on_commit(do_something)
 
-You can also wrap your function in a lambda::
+You can also bind arguments to your function using `functools.partial`_::
```

Comment:
We can use an intersphinx link here: 

```suggestion
You can also bind arguments to your function using :func:`functools.partial`::
```

And then remove the link below.

Reply:
Sweet, didn't know about intersphinx!